tests/e2e/testcases/upgrade/suite.go

```diff
@@ -56,6 +57,7 @@ var _ = ginkgo.Describe("[Upgrade]", ginkgo.Ordered, func() {
 		err = utils.DeleteConfigs(secondSubnetName)
 		gomega.Expect(err).Should(gomega.BeNil())
 		_ = utils.DeleteKey(keyName)
+		utils.DeleteCustomBinary(subnetName)
```

Comment:
Nit, and possibly overkill, but in tests like this it's often best practice to then check that the file actually really doesn't exist anymore.

Reply:
We use this pattern lots of places, I'm not very concerned about it